common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/FetchShuffleBlocks.java

```diff
@@ -42,10 +42,11 @@ public FetchShuffleBlocks(
       String appId,
       String execId,
       int shuffleId,
+      int shuffleSequenceId,
```

Comment:
All the changes in this class are backward incompatible. You just need to `FetchShuffleBlockChunks`